pandas/core/frame.py

```diff
@@ -1974,7 +1974,7 @@ def to_gbq(
         chunksize: int | None = None,
         reauth: bool = False,
         if_exists: str = "fail",
-        auth_local_webserver: bool = False,
+        auth_local_webserver: bool = True,
```

Comment:
this is not a deprecation but just a change. You can do that but update the whatsnew note and put it in other API changes.

Reply:
Done. Moved to "Other API changes"